wordclouds.py

```diff
@@ -0,0 +1,62 @@
+import json
+import os
+import matplotlib.pyplot as plt
+import numpy as np
+import re
+from wordcloud import WordCloud, STOPWORDS
+import nltk
+nltk.download('maxent_ne_chunker')
+nltk.download('words')
+nltk.download('punkt')
+nltk.download('averaged_perceptron_tagger')
+
+def wordcloud():
+    loc = input('Enter facebook archive extracted location: ')
+    if not os.path.isdir(loc):
+        print("The provided location doesn't seem to be right")
+        exit(1)
+    
+    fname = loc+'/comments/comments.json'
+    if not os.path.isfile(fname):
+        print("The file posts_and_commments.json is not present at the entered location.")
+        exit(1)
+
+    with open(fname) as f:
+        base_data = json.load(f)
+    
+    data = base_data["comments"]
```

Comment:
it would be better to check if the user has any data or not in the comments file (i.e. if he has any comments or not) and display a message (instead of an error) if the user has no comments.